src/sentry/eventstore/processing/base.py

```diff
@@ -23,9 +23,9 @@ class BaseEventProcessingStore:
     implementations.
     """
 
-    def __init__(self, inner: KVStorage[str, Event], timeout: int = DEFAULT_TIMEOUT):
```

Comment:
This was removed from the constructor since there is no consistent way to provide it as an option (e.g. the existing Redis Cluster implementation could not have been provided this option.)

If this is necessary, it'd make sense to split out the application configuration (options for event processing store, but _not_ which class to use) from the backend configuration (which backend class, and its options), but I don't see a compelling reason to do that right now.